### DISCUSSION TITLE
Theme submission: "jelly"

### DISCUSSION THREAD

**Discussion by stamminator** [JanDeDobbeleer/oh-my-posh#5383] [Themes]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

Just thought I'd submit my theme in case anyone could benefit from it. I started with the `emodipt-extend` theme, then heavily modified it for my use case. It's called `jelly`, and you can find it here: https://gist.github.com/stamminator/9be338d5312908bea33c90adafacfc17. 

![jelly-theme-ohmyposh](https://github.com/user-attachments/assets/df756706-f7ee-4df5-b092-93a413255390)

Notable features are:

- No uneven breaks/gaps seen in most powerline-based themes.
- Copy/paste-able file paths with no mid-path truncation.
- .NET version, Node.js version, and Git info
- If you're in an elevated prompt, you can't miss the red shield.

**Comment by mariluski** [comment]
cool

**Comment by abdigama** [comment]
so cool..

**Comment by ardalis** [comment]
how does one submit a theme for the gallery in the docs?

**Reply by JanDeDobbeleer** [reply]
@ardalis no longer possible.

**Reply by ardalis** [reply]
Ah, ok, thanks. Probably worth noting in the docs. I kept looking for it here:
https://ohmyposh.dev/docs/share

and later in the repo README.

Cheers.

Steve
